docs/contributing/assets/allowed_actions.json

```
@@ -9,6 +9,7 @@
         "v4.0.0",
         "v6.0.1",
         "v6.1.0",
+        "v6.1.1",
```

Please use the commit hash of the release

```suggestion
        "v6.1.1","971e284b6050e8a5849b72094c50ab08da042db8"
```